### PR TITLE
Add styling to admonitions

### PR DESCRIPTION
**Pull request by Mariatta**
- added box border and background color

This is to address #428 and python/peps#109

If `.. Note::` directive is used in page, then it will be rendered in a box

example screenshots:
for code of conduct page
screen shot 2016-09-26 at 9 24 48 pm

for pep 339
screen shot 2016-09-26 at 9 23 07 pm

### CODE REVIEW

**Comment by gvanrossum** [comment]
Nice, but shouldn't there be a little bit of a left margin inside the box?

**Comment by Mariatta** [comment]
Thanks for the feedback @gvanrossum.
I updated the code. These are updated screenshots.

screen shot 2016-09-27 at 9 07 25 am
screen shot 2016-09-27 at 9 07 04 am

**Comment by gvanrossum** [comment]
Perfect! I don't have commit access; maybe Better can merge?

File: static/sass/style.scss [hunk 1]
```diff
@@ -2161,6 +2161,16 @@ $colors: $blue, $psf, $yellow, $green, $purple, $red;
...
 }
 
+div.note {
+    background-color: #eee;
+    border: 1px solid #ccc;
+    padding: 5px 10px;
+}
+
+p.admonition-title {
+    margin: 0px 10px 5px 0px;
```
Comment (berkerpeksag) [inline static/sass/style.scss]: Right margin is not needed here. I'd just use `margin-bottom: 5px`.

Comment (berkerpeksag) [reply static/sass/style.scss]: Also, using `px` would break rendering in other devices. Please use the `px2em()` helper.

**Review by berkerpeksag — CHANGES_REQUESTED** [review]

**Comment by berkerpeksag** [comment]
It looks like `.text p` adds some bottom margin so it would be nice to add something like

``` css
div.note > p {
    margin-bottom: 0;
}
```

to make the white-space more balanced. This way you may also be able to remove the margin attribute in `p.admonition-title`.

**Comment by Mariatta** [comment]
Thanks @berkerpeksag. 
I use px2em() as suggested, and also set the bottom margin to 0

Here are the updated screenshots

screen shot 2016-09-27 at 9 49 22 am

screen shot 2016-09-27 at 9 49 17 am

**Review by berkerpeksag — APPROVED** [review]

**Comment by berkerpeksag** [comment]
Looks great, thanks!

**Pull request merged**